### PR TITLE
Kenya enrollment fixes

### PR DESCRIPTION
**Pull request by shejasabin**
* [SER-334](https://oneacrefund.atlassian.net/browse/SER-334) Kenya Enrollment/Just In time fixes(part 2)
    * updated maize bundle logic
    * updated eligibility to topUp/enroll check
    * added a back navigation

To test [here](https://telerivet.com/p/0c6396c9/service/SVc3c8d451e239d756/edit) 
**Test JiT enrollment:**

- enter a GL account number: **13983972**
- choose option 3 JiT Enrollment
- use an account number in the same group: **31217433** or **31217454** or **31217479**
- choose any product( can choose up to 3 products. a product ordered should not appear on the list next time. can only choose one quantity for maize bundles and can choose variety. back navigation also works)
- Finalize the order. ***the used account number can't order or top up the next time**

**To test TopUp**

- enter a GL account number: **13983972**
- choose option 4 JiT TopUp
- use an account number in the same group: **12122497** (only account enrolled in LR2021 can topUp and once)
- choose any product( can choose up to 3 products. a product ordered should not appear on the list next time. can only choose one quantity for maize bundles and can choose variety. back navigation also works)
- Finalize the order. ***the used account number can't order or top up the next time**

### CODE REVIEW

File: client-registration/bundle-choice-handler/bundleChoiceHandler.js [hunk 1]
```diff
@@ -34,6 +34,9 @@ module.exports = {
                 }
```
Comment (Bahatiroben) [inline client-registration/bundle-choice-handler/bundleChoiceHandler.js:34]: hey, can you add an else handler to take users back to the products menu? cause at the moment once you enter a wrong entry, eg: 60 or any other product not on the list, the session gets terminated

Comment (shejasabin) [reply client-registration/bundle-choice-handler/bundleChoiceHandler.js:33]: I have added it

**Comment by Bahatiroben** [comment]
> * [SER-334](https://oneacrefund.atlassian.net/browse/SER-334) Kenya Enrollment/Just In time fixes(part 2)
>   
>   * updated maize bundle logic
>   * updated eligibility to topUp/enroll check
>   * added a back navigation
> 
> To test [here](https://telerivet.com/p/0c6396c9/service/SVc3c8d451e239d756/edit)
> **Test JiT enrollment:**
> 
> * enter a GL account number: **13983972**
> * choose option 3 JiT Enrollment
> * use an account number in the same group: **31217433** or **31217454** or **31217479**
> * choose any product( can choose up to 3 products. a product ordered should not appear on the list next time. can only choose one quantity for maize bundles and can choose variety. back navigation also works)
> * Finalize the order. ***the used account number can't order or top up the next time**
> 
> **To test TopUp**
> 
> * enter a GL account number: **13983972**
> * choose option 4 JiT TopUp
> * use an account number in the same group: **12122497** (only account enrolled in LR2021 can topUp and once)
> * choose any product( can choose up to 3 products. a product ordered should not appear on the list next time. can only choose one quantity for maize bundles and can choose variety. back navigation also works)
> * Finalize the order. ***the used account number can't order or top up the next time**

So, @shejasabin what is supposed to happen if you try to enroll or top-up a client from a different group?

**Comment by shejasabin** [comment]
> > * [SER-334](https://oneacrefund.atlassian.net/browse/SER-334) Kenya Enrollment/Just In time fixes(part 2)
> >   
> >   * updated maize bundle logic
> >   * updated eligibility to topUp/enroll check
> >   * added a back navigation
> > 
> > To test [here](https://telerivet.com/p/0c6396c9/service/SVc3c8d451e239d756/edit)
> > **Test JiT enrollment:**
> > 
> > * enter a GL account number: **13983972**
> > * choose option 3 JiT Enrollment
> > * use an account number in the same group: **31217433** or **31217454** or **31217479**
> > * choose any product( can choose up to 3 products. a product ordered should not appear on the list next time. can only choose one quantity for maize bundles and can choose variety. back navigation also works)
> > * Finalize the order. ***the used account number can't order or top up the next time**
> > 
> > **To test TopUp**
> > 
> > * enter a GL account number: **13983972**
> > * choose option 4 JiT TopUp
> > * use an account number in the same group: **12122497** (only account enrolled in LR2021 can topUp and once)
> > * choose any product( can choose up to 3 products. a product ordered should not appear on the list next time. can only choose one quantity for maize bundles and can choose variety. back navigation also works)
> > * Finalize the order. ***the used account number can't order or top up the next time**
> 
> So, @shejasabin what is supposed to happen if you try to enroll or top-up a client from a different group?

It shouldn't allow them to continue

File: client-enrollment/clientEnrollment.test.js [hunk 2]
```diff
@@ -54,7 +58,7 @@ describe('clientRegistration', () => {
             clientEnrollment.start(account, country, enr_lang);
             expect(roster.getClient).toHaveBeenCalledTimes(1);
         });
-        it('should send a message if loan is fully paid',()=>{
+        xit('should send a message if loan is fully paid',()=>{
```
Comment (Bahatiroben) [inline client-enrollment/clientEnrollment.test.js:61]: can you check all test cases with **xit** and change them to **it**, I guess xit is supposed to skip them during the execution.

Comment (shejasabin) [reply client-enrollment/clientEnrollment.test.js:61]: These test some commented out codes for enrollment(which I think will be uncommented for the next season) then commented again when the JiT starts(didn't want to delete them)

Comment (shejasabin) [reply client-enrollment/clientEnrollment.test.js:61]: maybe later checking the current period first can help avoid this

Comment (Bahatiroben) [reply client-enrollment/clientEnrollment.test.js:61]: okay.

**Review by Bahatiroben — APPROVED** [review]

**Review by nyabongo — APPROVED** [review]

**Pull request merged**